upgrade/1.0.1/scripts/upgrade/add_pod_priority.sh

```diff
@@ -107,6 +99,14 @@ function wait_for_running_pods() {
   done
 }
 
+# Ensure etcd clusters have three running members before proceeding
+for etcdcluster in $ETCDCLUSTERS; do
+  ns=$(kubectl get etcdcluster -A | grep " $etcdcluster " | awk '{print $1}')
+  if [ ! -z "$ns" ]; then
+    wait_for_running_pods $ns $etcdcluster 3
```

Comment:
Could any customer customize (or scale up) etcd cluster members to be more than 3 (e.g., 5 or 7), therefore breaking the algorithm in this PR?

Reply:
great catch -- it looks like we're not going to merge this.  but if it gets resurrected i'll dynamically discover the desired size of the etcd cluster accounting for customizations...